test/src/tests-identity.ts

```diff
@@ -1932,14 +2536,19 @@ describe('identity', function() {
             null
         );
 
-        mockServer.respondWith(urls.login, [
-            200,
-            {},
-            JSON.stringify({ mpid: testMPID, is_logged_in: false }),
-        ]);
+        fetchMockSuccess(urls.login, {
+            mpid: testMPID,
+            is_logged_in: true,
+        });
 
         mParticle.Identity.login(identityAPIRequest1);
+
+        waitForCondition(() => mParticle.Identity.getCurrentUser().getMPID() === 'otherMPID')
+        .then(() => {
+
         fetchMock.resetHistory();
+
+        // QUESTION: Why are we logging a checkout of "1"?
```

Comment:
not sure.  this is technically a deprecated method also, so maybe we should log a checkout via logProductAction and see if it still works